apps/balancer-tools/src/app/apr/(utils)/calculatePoolStats.ts

```diff
@@ -3,11 +3,20 @@ import * as balEmissions from "#/lib/balancer/emissions";
 import { Pool } from "#/lib/balancer/gauges";
 import { pools } from "#/lib/gql/server";
 
-import { PoolStatsData } from "../api/route";
+import { PoolStatsData, PoolTokens } from "../api/route";
 import { getBALPriceByRound } from "./getBALPriceByRound";
 import { getPoolRelativeWeight } from "./getRelativeWeight";
 import { Round } from "./rounds";
 
+export enum PoolTypeEnum {
```

Comment:
😢 thought this would be generated by gql generator

Reply:
I think this makes sense since its suposed to a human readable, searching for it looks like Pool Simulator has a few variations of this, but none had the same output as the `app.balancer`. That's where I got the names btw, I'll add a comment saying that.